src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -183,6 +190,19 @@ AmendmentTableImpl::addInitial (Section const& section)
     }
 }
 
+void AmendmentTableImpl::addVetos (Section const& section)
+{
+    std::vector<AmendmentName> names;
+    parseSection (names, section);
+    for (auto const& n : names)
+    {
+        // Unknown amendments are effectively vetoed already
```

Comment:
We need a `ScopedLockType sl (mLock);` before calling `getExisting` since `getExisting` returns a pointer into the `m_amendmentMap`.